locales/en-US/app.ftl

```diff
@@ -121,6 +121,7 @@ CallNodeContextMenu--searchfox = Look up the function name on Searchfox
 CallNodeContextMenu--copy-function-name = Copy function name
 CallNodeContextMenu--copy-script-url = Copy script URL
 CallNodeContextMenu--copy-stack = Copy stack
+CallNodeContextMenu--copy-call-stack = Copy call stack
```

Comment:
I think we can remove this as we don't need another "copy stack"-like context menu items. See my review comment.